README.md

```diff
@@ -1,2 +1,80 @@
-# otel
-OpenTelemetry instrumentation library.
+# @fastify/otel
+
+[![CI](https://github.com/fastify/otel/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/fastify/otel/actions/workflows/ci.yml)
+
+<!-- [![NPM version](https://img.shields.io/npm/v/@fastify/otel.svg?style=flat)](https://www.npmjs.com/package/fastify-plugin) -->
+
+[![neostandard javascript style](https://img.shields.io/badge/code_style-neostandard-brightgreen?style=flat)](https://github.com/neostandard/neostandard)
+
+`fastify-plugin` is a plugin helper for [Fastify](https://github.com/fastify/fastify).
+
+OpenTelemetry auto-instrumentation library.
```

Comment:
I don't understand what this is trying to convey. Why are we talking about `fastify-plugin`?

Reply:
I think is showing some outdated version, mine says `@fastify/otel`, but let me see if a push fixes it (TL;DR took the `README` from `fatify-plugin` to follow the format)